app/views/ReportsView/index.tsx

```diff
@@ -67,7 +67,7 @@ const ReportsTableComponent = (): JSX.Element => {
       };
       getData();
     }
-  }, [adminAccess, isExternalMode, rowData]);
+  }, [adminAccess, nonproductionAccess, unreviewedAccess, rowData]);
```

Comment:
missing dependencies here (allStates, nonproductionStates, and unreviewedStates)